index.js

```diff
@@ -0,0 +1,15 @@
+"use strict"
+
+module.exports = {
+  extends: [
+    "eslint:recommended",
```

Comment:
Rationale for skipping `"airbnb", "airbnb-typescript"` in this list?

Reply:
Here db111be I've added `airbnb` support for typescript linting. I didn't notice `airbnb-typescript` at first and thought that in terms of `.js` files we could run into conflicts, cause at least in some `keep-*` projects we use `google` standard. Maybe it's a good idea to standardize linting across all `Thesis` projects.